.env.example

```
@@ -82,4 +82,4 @@ META_VERIFY_ATTEMPTS=3
 # FALLBACK_META_AD_ACCOUNT_ID  - Fallback Meta Ad Account (plain, required secret)
 FALLBACK_META_AD_ACCOUNT_ID=
 SHEETS_WEBHOOK_URL_DOCTORALIA=
-SHEETS_WEBHOOK_SECRET_DOCTORALIA=Doctoralia_Secret_2026_!!
+SHEETS_WEBHOOK_SECRET_DOCTORALIA=
```

dotenv-linter (4.0.0)

[warning] [DuplicatedKey] The SHEETS_WEBHOOK_SECRET_DOCTORALIA key is duplicated

(DuplicatedKey)